src/pkg/backup/details/details.go

```diff
@@ -127,7 +128,13 @@ func (dm DetailsModel) FilterMetaFiles() DetailsModel {
 // additional data like permissions in case of OneDrive and are not to
 // be treated as regular files.
 func (de DetailsEntry) isMetaFile() bool {
-	return de.ItemInfo.OneDrive != nil && de.ItemInfo.OneDrive.IsMeta
+	if de.ItemInfo.OneDrive == nil {
+		return false
+	}
+
+	return de.ItemInfo.OneDrive.IsMeta ||
+		strings.HasSuffix(de.RepoRef, ".meta") ||
+		strings.HasSuffix(de.RepoRef, ".dirmeta")
```

Comment:
nit should use OneDrive consts if possible

Reply:
Nevermind!  Circular imports there.

Reply:
hmmm alright. Thanks for testing it out. I think maybe it will be cleaned up in a later patch